scripts/maint/check_subsys_ordering.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/python3
+
+from __future__ import print_function
+import re
+import sys
+
+# Not every subsystem is initialized in a place corresponding to its
+# name.  These are the locations of the ones that are not.
+LOCATIONS = {
+    "btrack" : "feature/control",
+    "network" : "lib/net",
+    "ocirc_event" : "core/or",
+    "orconn_event" : "core/or",
```

Comment:
I thought you removed orcirc_event and orconn_event in an earlier commit?

Reply:
ugh, I believe so.  Also this entire script should be junked: the new function in includes.py is what we should be using instead.